packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js

```diff
@@ -324,6 +324,64 @@ describe("react-wildcat-prefetch", () => {
                 ReactTestUtils.renderIntoDocument(<WrappedPrefetch title={"Test Title"} />);
             });
 
+            it("hydrates multiple React components", (done) => {
+                let renderCount = 1;
+
+                class FirstHydrationTest extends React.Component {
+                    static propTypes = {
+                        firstData: React.PropTypes.object
+                    }
+
+                    static defaultProps = {
+                        firstData: {}
+                    }
+
+                    render() {
+                        expect(this.props.firstData)
+                            .to.eql(stubs.prefetchedData.firstData);
+
+                        if (renderCount > 1) {
+                            done();
+                        }
```

Comment:
That's not what's happening here. Each component is rendering once, but I'm only calling `done()` when both have rendered.